fw/http_frame.c

```diff
@@ -1878,7 +1878,9 @@ tfw_h2_frame_process(TfwConn *c, struct sk_buff *skb)
 		h2->data_off = 0;
 		h2->skb_head = data_up.skb->next = data_up.skb->prev = NULL;
 		r = tfw_http_msg_process_generic(c, h2->cur_stream, data_up.skb);
-		if (r == T_DROP) {
+		/* TODO Check this place, when working on #1490 */
```

Comment:
Not a big deal, but we usually we write such comments like `/* TODO #1490: Check this place, when working on */`